src/sass/components/_repo-images.scss

```diff
@@ -0,0 +1,42 @@
+// Repo Images
+
+// Variables specific to repo images
+$repo-img-size: calc(3.5rem + 3vw);
+$repo-img-size-static: 4.75rem;
+$repo-img-pos-adjust: -1.125rem;
+
+.ms-repo-item {
+  // increase default height/weight of icon in repo visualpicker
+  .slds-media__figure {
+    > div {
+      height: $repo-img-size;
+      width: $repo-img-size;
+
+      .slds-avatar_large {
+        height: 100%;
+        width: 100%;
+        font-size: $font-size-10;
+        font-weight: $font-weight-bold;
```

Comment:
For "Fallback Entity Initials" I assume? Do we bold them elsewhere? That seems like a less necessary change to  SLDS, but ok.

Reply:
Good point, Salesforce intentionally lists font-weight: 300 so I will remove that.